bins/packages/storcli/storcli.sh

```diff
@@ -0,0 +1,32 @@
+dependencies_storcli() {
+    apt-get update
+    apt-get install wget
```

Comment:
`wget` is already installed by basesystem, which is supposed to be installed first for every packages.
You can skip this part.